notice_and_comment/static/regulations/css/less/module/nc-homepage-custom.less

```diff
@@ -223,9 +223,14 @@ Read and write with ease
         .cf-icon {
           margin-left: auto;
         }
+
+        .button-close {
+          display: none;
+        }
       }
 
       .collapsible {
+        display: none;
```

Comment:
Same concern as in eregs/regulations-site#352 - this prevents the text from being visible for non-JS users.